seeds/breadboard/docs/endpoint-api.md

```diff
@@ -1,3 +0,0 @@
-# Breadboard Endpoint API Documentation
```

Comment:
ah -- we probably need to move this somewhere. I don't want to delete this file.

Reply:
Where should i move it to?

Reply:
Oops, please disregard. This was a comment from earlier in the review process where I didn't realize that we needed to move the generated docs to `docs/api`.